### PR TITLE
Improve summary page responsiveness

### PR DESCRIPTION
**Pull request by hugonxc**
This is my collaboration related to issue #134, there you can see the previous behavior of the summary page. Now, these are the behaviors:
On the desktop view:
newdle_after
And the mobile view:
newdle_mobile_after
As you can see, I change the logo when the site goes to a responsive view. I made this image, which is in PNG format, by my self(I'm not a professional designer) so it's recommended to change it later by your official design pattern.

All tests run successfully.

I hope this PR can collaborate on this project.

### CODE REVIEW

File: newdle/migrations/versions/20191024_1629_000000000000_create_initial_tables.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 """Create initial tables
 
 Revision ID: 000000000000
-Create Date: 2019-09-12 17:18:07.401658
+Create Date: 2019-10-24 16:29:39.913975
```
Comment (mic4ael) [inline newdle/migrations/versions/20191024_1629_000000000000_create_initial_tables.py]: This shouldn't end up in the commit

File: newdle/client/src/components/TopHeader.module.scss [hunk 2]
```diff
@@ -8,3 +8,11 @@
     height: 2.5em;
   }
 }
+
+.logo {
+  content: url('../images/logo.svg');
+
+  @media (max-width: 768px) {
+    content: url('../images/logo_mobile.png');
```
Comment (ThiefMaster) [inline newdle/client/src/components/TopHeader.module.scss]: Why do we need a png for this? Shouldn't it be possible to just scale the SVG properly?

Comment (pferreir) [reply newdle/client/src/components/TopHeader.module.scss]: Shouldn't this be solved by #144 anyway? Maybe this can just be removed.

Comment (ThiefMaster) [reply newdle/client/src/components/TopHeader.module.scss]: Indeed, please rebase to the latest master to have all the responsive fixes related to the front page and global elements.

**Comment by hugonxc** [comment]
If I use unstackable this is the result on mobile and desktop:
Desktop:
desk_unstack
Mobile
unstackable

Do you prefer this way?

**Comment by ThiefMaster** [comment]
Please use rebases instead of merges to update your branch; otherwise we need to squash the commits when merging to avoid the mess merge commits in PR branches bring with them.

**Comment by ThiefMaster** [comment]
About the UI, it looks a bit "squeezed" to me in the mobile view. BTW, make sure to submit some availability for the newdle to see how it looks like with participants being listed there.

**Pull request closed without merging**